tests/unit/test_edge.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from localstack.services.edge import is_s3_form_data
+
+
+class EdgeServiceTest(unittest.TestCase):
+
+    def data_contains_key_equal_is_true(self):
```

Comment:
Are these test methods actually executed when we run `make test`? My understanding is that only methods with name pattern `test_xxx` get executed by default.

Reply:
Oops, that'll teach me to be lazy and forget to test without `TEST_PATH` before pushing! Fixed it, thanks :)